### PR TITLE
Made little changes 

### PR DESCRIPTION
**Pull request by UceeCode**
- I increased the max-width for the about us text
- I added a waving hand emoji animation
- I also styled the header section for mobile screen

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -53,7 +54,7 @@ body{
 
 
 .links a{
-    margin-left: 12px;
+    margin-left: 17px;
```
Comment (dokasto) [inline css/style.css]:
```suggestion
    margin-left: 16px; // or 20px remember multiple of 4
```

File: css/style.css [hunk 2]
```diff
@@ -28,6 +28,7 @@ body{
 
 .main .header {
     width: 100%;
+    padding: 10px;
```
Comment (dokasto) [inline css/style.css]:
```suggestion
    padding: 12px; // or 16px remember multiple of 4
```

File: css/style.css [hunk 3]
```diff
@@ -78,9 +79,30 @@ body{
     font-size: 22px;
     line-height: 1.2;
     font-family: sans-serif;
-    max-width: 400px;
+    max-width: 700px;
+    margin-bottom: 15px;
```
Comment (dokasto) [inline css/style.css]:
```suggestion
    margin-bottom: 16px;
```

File: css/style.css [hunk 4]
```diff
@@ -147,3 +169,29 @@ body{
     margin-top: 12px;
 } 
 
+@media (max-width: 768px){
+    .header h1{
+        position: relative;
+        left: 12px;
+    }
```
Comment (dokasto) [inline css/style.css]: why not use margin? 
Don't use `top` or `bottom` or `left` or `right` unless you position an element absolutely. 

```suggestion
    .header h1{
        margin: 12px;
    }
```

File: css/style.css [hunk 5]
```diff
@@ -147,3 +169,29 @@ body{
...
+    .links a {
+        font-size: 14px;
+        position: relative;
+        top: 15px;
```
Comment (dokasto) [inline css/style.css]:
```suggestion
        font-size: 14px;
        margin-top: 16px;
```

File: css/style.css (hunk 1 above)
Comment (UceeCode) [reply css/style.css]: done 👍

File: css/style.css (hunk 2 above)
Comment (UceeCode) [reply css/style.css]: done ☑️

File: css/style.css (hunk 3 above)
Comment (UceeCode) [reply css/style.css]: done 👍

File: css/style.css (hunk 4 above)
Comment (UceeCode) [reply css/style.css]: Done 👍

File: css/style.css (hunk 5 above)
Comment (UceeCode) [reply css/style.css]: done 👍

**Pull request closed without merging**